slurm/02_run_scenarios.sh

```
@@ -0,0 +1,83 @@
+# Run all scenario analyses with independent GT and INC variation
+# Array structure: 432 jobs = 3 diseases × 4 scenario_types × 6 GT × 6 INC
+# Limited to 40 concurrent jobs (LSHTM max)
+#
+# Per disease: 144 jobs (4 scenarios × 36 GT×INC combinations)
+# Jobs 1-144:    ebola
+# Jobs 145-288:  covid
+# Jobs 289-432:  cholera
+
+# Get project root from SLURM_SUBMIT_DIR
+if [[ "$SLURM_SUBMIT_DIR" == */slurm ]]; then
+    cd "$SLURM_SUBMIT_DIR/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)